code/go/0chain.net/smartcontract/storagesc/blobber_integration_tests.go

```diff
@@ -13,13 +14,13 @@ import (
 )
 
 // insert new blobber, filling its stake pool
-func (sc *StorageSmartContract) insertBlobber(t *transaction.Transaction,
+func (ssc *StorageSmartContract) insertBlobber(t *transaction.Transaction,
```

Comment:
Please avoid unrelated refactoring. Not only can it lead to hard to find errors and obscure the real changes being made, but it can also make merging the PR a nightmare.

I cannot see any real changes in blobber_integration_tests.go, if so can you revert these changes.